docs/rewind/release_notes.md

```diff
@@ -1,10 +1,10 @@
 ---
-sidebar_position: 13
+sidebar_position: 3
 ---
 
 # Release Notes
 
-Download the latest version of Fourier Rewind [here](https://fourieraudio.com/support).
+Download the latest version of Fourier Rewind [here](https://downloads.fourieraudio.com/rewind/latest/FourierRewind-Latest.zip).
```

Comment:
As above, might be worth mentioning the platform

Reply:
Done :)